source/bc/mus_bc_general_module.fpp

```diff
@@ -700,6 +701,8 @@ contains
         bc( iBnd )%fnct => inlet_pasScal
       case('flekkoy_outlet')
         bc( iBnd )%fnct => outlet_pasScal
+      case('pressure_antiBounceBack_pasScal')
+        bc( iBnd )%fnct => pressure_antiBounceBack_pasScal
```

Comment:
No need to introduce `'pressure_antiBounceBack_pasScal'` as boundary kind. Use existing `'pressure_antiBounceBack'` and select passive scalar routine using scheme kind.

Suggestion:
```
select case(trim(schemeHeader%kind))
case('fluid')
   bc( iBnd )%fnct => pressure_antiBounceBack
case('passive_scalar')
   bc( iBnd )%fnct => pressure_antiBounceBack_pasScal
case default
  call tem_abort('Unknown scheme kind for pressure_antiBounceBack')
end select
```